nilearn/decoding/tests/test_searchlight.py

```diff
@@ -72,3 +72,29 @@ def test_searchlight():
     sl.fit(data_img, cond)
     assert_equal(np.where(sl.scores_ == 1)[0].size, 33)
     assert_equal(sl.scores_[2, 2, 2], 1.)
+
+    # group cross validation
+    try:
+        from sklearn.model_selection import LeaveOneGroupOut
+        gcv = LeaveOneGroupOut()
+    except ImportError:
+        # won't import model selection if it's not there.
+        # the groups variable should have no effect.
+        gcv = cv
+
+    groups = np.random.permutation(np.arange(frames, dtype=int) >
+                                   (frames // 2))
+    sl = searchlight.SearchLight(mask_img, process_mask_img=mask_img, radius=1,
+                                 n_jobs=n_jobs, scoring='accuracy', cv=gcv)
+    sl.fit(data_img, cond, groups)
+    assert_equal(np.where(sl.scores_ == 1)[0].size, 7)
+    assert_equal(sl.scores_[2, 2, 2], 1.)
+
+    # adding superfluous group variable
```

Comment:
The results should be identical to the test in lines 57-67. The only difference is in sklearn >= 0.18 this will try LeaveOneGroupOut (cv=gcv) in line 88 rather than the KFold declared above. Let me know if I'm misunderstanding.